projects/js-packages/components/components/record-meter-donut/index.tsx

```diff
@@ -40,21 +54,35 @@ const DonutMeter: React.FC< DonutMeterProps > = ( {
 	backgroundColor = '#00BA37', // jetpack green fallback
 	thickness = '3.5',
 	donutWidth = '64px',
+	title = '',
+	description = '',
 } ) => {
 	const count = () => {
 		// get count as a percent value
 		return ( segmentCount / totalCount ) * 100;
 	};
 
+	// If we don't have a title or description, hide the meter from screen readers.
+	const isHidden =
+		typeof title === 'string' &&
+		title.length === 0 &&
+		typeof description === 'string' &&
+		description.length === 0
+			? 'true'
+			: 'false';
```

Comment:
Do you think it'd ever make sense to allow showing the a11y markup if only one of those props is passed rather than needing both?

Reply:
Hey @samiff. Thanks for the feedback. That's actually how it works. The markup is only hidden if both strings are empty/null. If one is present, the a11y markup is included. You can test that behaviour via Storybook to confirm.